### DISCUSSION TITLE
Certificate error

### DISCUSSION THREAD

**Discussion by auzziemale** [kubilus1/autoortho#103] [Q&A]
Hi guys I am running garuda linux. When trying to run autoortho_lin.bin i get this error.
urllib.error.URLError: <urlopen error [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:997)>
Can anybody help me I have looked on the net and dont seem to e able to fix it

**Comment by ivotaHR** [comment]
check you pc date - time.

**Reply by auzziemale** [reply]
its the correct time for me

**Comment by kubilus1** [comment]
Your system ca certs are likely out of date.  Not sure what the process is for your distro, but update those.

**Reply by auzziemale** [reply]
Done that as well. God knows what it is. I’m not fantastic in linux

**Reply by auzziemale** [reply]
I think you can disable the check if you know what the address it is looking for? Can you supply me the address?

**Comment by ylf-3b** [comment]
Got the same Error ...

**Comment by ylf-3b** [comment]
Now I made a fresh Install of the recommended Ubuntu 22.4 LTS on a separate drive and get autoortho to run.
To figure out, what's going wrong I compare the versions of certifi:
my Fedora 38:
Name: certifi
**Version: 2022.9.24**
Summary: Python package for providing Mozilla's CA Bundle.
Home-page: https://github.com/certifi/python-certifi
Author: Kenneth Reitz
Author-email: me@kennethreitz.com
License: MPL-2.0
Location: /usr/lib/python3.11/site-packages
Requires: 
Required-by: 

Ubuntu 22.04:
Name: certifi
**Version: 2020.6.20**
Summary: Python package for providing Mozilla's CA Bundle.
Home-page: https://certifiio.readthedocs.io/en/latest/
Author: Kenneth Reitz
Author-email: me@kennethreitz.com
License: MPL-2.0
Location: /usr/lib/python3/dist-packages
Requires: 
Required-by: 

So, I'm not a software developer, but for me, as user Updating the Certification can't be the solution or are there other certification packages relevant?

**Comment by guilhermecardozo** [comment]
Does anyone was successful to fix this? I'm having the same issue. I will try to change to Ubuntu or Mint again but I'm more comfortable using Fedora instead

**Reply by ylf-3b** [reply]
The version 0.4.3 and 0.5.0 starts without certification error on my system, but I still haven't tested it in combination running with x-plane together.

**Comment by kubilus1** [comment]
I think Nuitka builds in whatever the system certs are in the build image.  So going to try to update the certs in CI and see.  Take a look at the latest packages once they finish building.

**Comment by SamPro910** [comment]
Try to run AutoOrtho with `SSL_CERT_DIR=/etc/ssl/certs ./autoortho_lin.bin`.
